docs/spark-queries.md

```diff
@@ -143,6 +145,22 @@ spark.read
     .load("path/to/table")
 ```
 
+```scala
+// time travel to tag historical-snapshot
```

Comment:
[question] should we just say `read the table from the snapshot, historical-snapshot is pointing to` as it's not exactly a time travel what we are doing here, thoughts ?